manticore/platforms/evm.py

```diff
@@ -21,6 +22,8 @@
 TT255 = 2 ** 255
 TOOHIGHMEM = 0x1000
 
+PendingTransaction = namedtuple("PendingTransaction", ['sort', 'address', 'origin', 'price', 'data', 'caller', 'value', 'return_data', 'result'])
```

Comment:
i think some of these field names are not quite correct. if you look at the line where this tuple is destructured (https://github.com/trailofbits/manticore/blob/master/manticore/platforms/evm.py#L2306), we can see that `sort` -> `type`, `return_data` -> `bytecode`, `result` -> `header` would be more appropriate names :)

Reply:
I followed the field names used in the definition of Transaction at https://github.com/trailofbits/manticore/blob/master/manticore/platforms/evm.py#L31

Reply:
that is a good idea! but the elements of a PendingTransaction do not exactly correspond to the constructor arguments of a Transaction :)